### DISCUSSION TITLE
is there a way to render a marp presentation with hugo?

### DISCUSSION THREAD

**Discussion by simoneggler** [marp-team/marp#384] [Q&A] [answered]
Hi, I am using hugo to generate static content from markdown files. Is there a way to use hugo (https://gohugo.io/) to render the presentations in html?

**Comment by yhatt** [comment, marked as answer]
Maintainer does not familiar to Hugo and Golang so I don't know how to integrate Marp with Hugo. Asking in the Hugo community might get a clearer answer.

In general, you can use Marp Core upon Node.js to convert Markdown to HTML, and put its output to Hugo's output for getting seamless integration with Hugo's Markdown. Using [Hugo Pipes](https://gohugo.io/hugo-pipes/introduction/) might become able to use Node.js module resources including Marp.
